documentation/modules/auxiliary/scanner/mssql/mssql_hashdump.md

```diff
@@ -0,0 +1,91 @@
+## Description
+
+The `mssql_hashdump` module queries an MSSQL instance or session and returns hashed user:pass pairs. These pairs can be decripted via or `hashcat`.
+
+## Available Options
```

Comment:
Not a blocker; I don't think this follows the format that's expected for module documentation: https://github.com/rapid7/metasploit-framework/pull/19054#issuecomment-2065267268